templates/tutor.hbs

```diff
@@ -1,21 +1,21 @@
 {{! Maintainers - Kevin Corry, Daryl Hedley}}
-<div class="tutor" role="dialog" aria-label="{{_accessibility._ariaLabels.feedbackPopUp}}">
+<div class="tutor" role="region" aria-label="{{_globals._accessibility._ariaLabels.feedbackPopUp}}">
     <div class="tutor-toolbar">
```

Comment:
Need to move 

```
<div class="tutor-toolbar">
        <a href="#" class="tutor-done" role="button" aria-label="{{_globals._accessibility._ariaLabels.closePopup}}">
            <span class="tutor-icon-close icon icon-cross"></span>
         </a>
     </div>
```

to the bottom of the DOM order and style up at the top accordingly with CSS.